code/modules/mining/mine_items.dm

```diff
@@ -580,10 +580,10 @@
 	return ((locate(/obj/structure/track) in destination)) ? ..() : FALSE
 
 /obj/item/key/minecarts
-	name = "key"
+	name = "minecart key"
```

Comment:
Why would it be named minecart key instead of just key? The desc should say what the key is or describe it, not the name

Reply:
It's a key that clearly has a fob attached to it that identifies its purpose. There is no way to possibly mix it up with other keys on a visual level. Should we rename all dog breeds to just "dog?" And tell people to click on it to figure out what the dog breeds is? 

This is got to be one of the most pointlessly nitpicky change requests. I can get arguing about keeping the memes in because you have a stake in that emotionally. But why are you arguing for a genericness of key names? What is your goal?